tests/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixerTest.php

```diff
@@ -181,6 +181,38 @@ function hello($name)
 ',
                 ['annotations' => ['noinspection']],
             ],
+
+            [
+                '<?php
+/**
+* @internal
+* @AuThOr Jane Doe
+*/
+function foo() {}',
+                '<?php
+/**
+* @internal
+* @author John Doe
+* @AuThOr Jane Doe
+*/
+function foo() {}',
+                ['annotations' => ['author']],
```

Comment:
```suggestion
                ['annotations' => ['author'], 'case_sensitive' => false],
```

just to be explicit about config we test here, so we don't "adjust the code" when we change default configuration